react/components/Table/SwitchablePagination.js

```diff
@@ -15,8 +15,16 @@ SwitchablePagination.defaultProps = {
 }
 
 SwitchablePagination.propTypes = {
-  ...Pagination.propTypes,
```

Comment:
Where was it required? ```remove propTypes required in `SwitchablePagination`.```.

Reply:
the `currentItemFrom`, `currentItemTo`, `textOf`, `textShowRows`, `totalItems` props are required in pagination, but should not in `SwitchablePagination`.

Reply:
and I can't remove the required ones inside the required component, because there it makes sense to be. So the spread on all propTypes of the pagination does not make sense for SwitchablePagination.

Reply:
that's why I added here only the props that are actually used.

Reply:
Ok.